### ISSUE TITLE
Jinja2 Template

### ISSUE THREAD

**Issue by vstoyanoff** [lenarother/django-critical-css#6]
Hi,

I am quite new to Python and Django in particular and really want to make use of this amazing package. The project is set up with jinja templates and I can't seem to find a way to use the template tags or a way to import the functionality and use it with custom template tag. I will appreciate some help on this one. Thanks!

**Comment by lenarother** [comment]
Hi @vstoyanoff ,

Have you tried the solutions given in the Django documentation:
https://docs.djangoproject.com/en/3.1/topics/templates/#django.template.backends.jinja2.Jinja2 ?

Best,
Magda

**Comment by vstoyanoff** [comment]
Hey @lenarother,

Thanks for your suggestion. I was able to make it work with a custom tag and importing the function from the package. I searched the source code and found what I need. I currently experience a different problem (which might not be related). The service for extracting the CSS suggested in the docs is working when I manually test it, but when using the plugin it throws an error. Not sure what is happening. Is it possible to have a page reload for some reason?

Nevertheless thank you for this solution! It is awesome.

Best,
V.
